webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -3142,7 +3142,7 @@ var userOverride = {
                             );
                             kmChatInput.setAttribute(
                                 'oninvalid',
-                                'setCustomValidity(' + MCK_LABELS['lead.collection'].errorEmail + ')'
+                                `setCustomValidity('${ MCK_LABELS['lead.collection'].errorEmail}')`
```

Comment:
avoid using template strings in web-sdk
these are not supported in IE

Reply:
@Prochnost55 Done sir.